databuilder/query_engines/query_model_convert_to_old.py

```diff
@@ -165,9 +165,12 @@ def convert_combine_as_set(node: new.AggregateByPatient.CombineAsSet):
 
 def convert_aggregation(node):
     function = AGGREGATE_MAP[node.__class__]
-    assert isinstance(node.source, new.SelectColumn)
-    source_table = convert_node(node.source.source)
-    input_column = column_name(node.source.name)
+    if not isinstance(node.source, new.SelectColumn):
```

Comment:
I think at least a comment would be in order here to the effect that this is a `Frame`. Or maybe an explicit type test for the `Frame`.